diff --git a/base/cvd/build_external/mtools/config.h b/base/cvd/build_external/mtools/config.h
index 1de1c47..b94633e 100644
--- a/base/cvd/build_external/mtools/config.h
+++ b/base/cvd/build_external/mtools/config.h
@@ -323,7 +323,7 @@
 #define HAVE_TERMIOS_H 1
 
 /* Define to 1 if you have the <termio.h> header file. */
-#define HAVE_TERMIO_H 1
+#/* #undef HAVE_TERMIO_H -- removed: Fedora glibc >= 2.39 dropped <termio.h> */
 
 /* Define to 1 if you have the <time.h> header file. */
 #define HAVE_TIME_H 1
