diff --git a/base/cvd/MODULE.bazel b/base/cvd/MODULE.bazel
index 4d602e0..d9512e7 100644
--- a/base/cvd/MODULE.bazel
+++ b/base/cvd/MODULE.bazel
@@ -31,7 +31,7 @@ bazel_dep(name = "libjpeg_turbo")
 bazel_dep(name = "libpng", version = "1.6.47.bcr.1")
 bazel_dep(name = "libuuid")
 bazel_dep(name = "libxml2", version = "2.13.5")
-bazel_dep(name = "libzip", version = "1.10.1")
+bazel_dep(name = "libzip", version = "1.10.1.bcr.1")
 bazel_dep(name = "lz4", version = "1.9.4.bcr.2")
 bazel_dep(name = "nasm", version = "2.16.03.bcr.1")
 bazel_dep(name = "pcre2", version = "10.45")
