diff --git a/base/stamp_helper.sh b/base/stamp_helper.sh
index badec7d..ccea87b 100755
--- a/base/stamp_helper.sh
+++ b/base/stamp_helper.sh
@@ -13,4 +13,4 @@
 #   actions
 
 echo "STABLE_HEAD_COMMIT $(git rev-parse HEAD)"
-echo "STABLE_CF_COMMON_VERSION $(dpkg-parsechangelog --file ../debian/changelog --show-field Version)"
+echo "STABLE_CF_COMMON_VERSION $(head -1 ../debian/changelog | sed 's/.*(\(.*\)).*/\1/')"
