diff --git a/base/cvd/MODULE.bazel b/base/cvd/MODULE.bazel
index 4d602e0..649e91d 100644
--- a/base/cvd/MODULE.bazel
+++ b/base/cvd/MODULE.bazel
@@ -17,7 +17,7 @@ bazel_dep(name = "cxx.rs", version = "1.0.194")
 bazel_dep(name = "fmt", version = "11.2.0.bcr.1")
 bazel_dep(name = "freetype", version = "2.13.3.bcr.2")
 bazel_dep(name = "gflags")
-bazel_dep(name = "googleapis", version = "0.0.0-20241220-5e258e33.bcr.1")
+bazel_dep(name = "googleapis", version = "0.0.0-20251003-2193a2bf")
 bazel_dep(name = "googleapis-cc", version = "1.0.0")
 bazel_dep(name = "googletest")
 bazel_dep(name = "grpc")
@@ -36,13 +36,13 @@ bazel_dep(name = "lz4", version = "1.9.4.bcr.2")
 bazel_dep(name = "nasm", version = "2.16.03.bcr.1")
 bazel_dep(name = "pcre2", version = "10.45")
 bazel_dep(name = "platforms", version = "1.0.0")
-bazel_dep(name = "protobuf", version = "31.1")
+bazel_dep(name = "protobuf", version = "33.0")
 bazel_dep(name = "rootcanal")
 bazel_dep(name = "rules_cc", version = "0.2.16")
 bazel_dep(name = "rules_flex", version = "0.4")
 # this is normally an indirect dependency, but for bazel 9 we need a higher version
 bazel_dep(name = "rules_foreign_cc", version = "0.15.1")
-bazel_dep(name = "rules_java", version = "7.12.2")
+bazel_dep(name = "rules_java", version = "8.16.1")
 bazel_dep(name = "rules_license", version = "1.0.0")
 # this is normally an indirect dependency, but for bazel 9 we need a higher version
 bazel_dep(name = "rules_nodejs", version = "6.7.3")
